Bricks/util/random.h

```diff
@@ -103,4 +103,14 @@ inline double RandomDouble(const double a, const double b) { return RandomReal<d
 }  // namespace random
 }  // namespace bricks
 
+using bricks::random::RandomInt;
+using bricks::random::RandomLong;
+using bricks::random::RandomInt64;
```

Comment:
Perhaps the other way around: retire `Int` and `Long` versions, leaving only the ones tied to byte-count-specific types?